### ISSUE TITLE
Feat: Remove /user/me usage in frontend

### ISSUE THREAD

**Issue by inkhey** [tracim/tracim#3090]
## Feature description

After discussion with @Skylsmoi in https://github.com/tracim/tracim/issues/3051,
it seems better for frontend to not use /user/me endpoints in order to limit the number of request done by frontend.

**Comment by Skylsmoi** [comment]
is the "number of requests" only related to the redirections of http 307 ?

**Comment by inkhey** [comment]
> is the "number of requests" only related to the redirections of http 307 ?

Yes, with redirection 307, you get 2 request instead of one because the first request return 307.
